wb-codemod/package.json

```diff
@@ -1,10 +1,18 @@
 {
   "name": "@khanacademy/wb-codemod",
-  "version": "2.1.1",
   "license": "MIT",
+  "version": "2.1.1",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Khan/wonder-blocks.git",
+    "directory": "wb-codemod"
```

Comment:
Trusted Publishing requires that the `repository` section be filled out like this.